nixos/modules/programs/ssmtp.nix

```diff
@@ -45,6 +47,21 @@ in
         '';
       };
 
+      settings = mkOption {
+        type = with types; attrsOf (oneOf [ bool str ]);
+        default = {};
+        description = ''
+          ssmtp configuration. Refer to
```

Comment:
totally optional, but to reference a man-page in docbook, you can also do something like this: `<citerefentry><refentrytitle>ssmtp</refentrytitle><manvolnum>5</manvolnum></citerefentry>`.

Reply:
What a wonderful suggestion. Makes me wish I knew this documentation format better. I'll definitely add this.

Reply:
@Ma27 I feel like this isn't rendering correctly... Any thoughts?
```
      settings = mkOption {
        type = with types; attrsOf (oneOf [ bool str ]);
        default = {};
        description = ''
          <citerefentry><refentrytitle>ssmtp</refentrytitle><manvolnum>5</manvolnum></citerefentry> configuration. Refer
          to <link xlink:href="https://linux.die.net/man/5/ssmtp.conf"/> for details on supported values.
        '';
        example = literalExample ''
          {
            Debug = true;
            FromLineOverride = false;
          }
        '';
      };
```

![Screenshot_20200607_103333](https://user-images.githubusercontent.com/7755101/83971655-6cb61380-a8aa-11ea-9a4c-a10d50de4d43.png)

Reply:
> Makes me wish I knew this documentation format better. I'll definitely add this.

I recommend https://docbook.rocks/ 

> I feel like this isn't rendering correctly... Any thoughts?

Sorry, I wasn't clear enough. This is atm just a semantic measure that doesn't have any useful formatting unfortunately.

I still think this is useful if we ever manage to write something which properly linkifies man-pages in our manual (in fact it's somewhere on my todo list unless someone is faster :p)

Reply:
Thanks for the info!